client/testFixture/SampleCode.sas

```diff
@@ -52,3 +52,8 @@ proc univariate data=popstate freq plot normal;
 proc print data= univout;
 	title 'Output Dataset From PROC UNIVARIATE';
 	format popn50 pop_50 pop_95 pop_97_5 pop_100 best8.;
+proc iml;
```

Comment:
This sample code is used in various places. I would suggest to not change it here. The specific test can edit active editor on the fly.